lib/file-view.coffee

```diff
@@ -19,7 +19,11 @@ class FileView extends HTMLElement
     @fileName.dataset.name = @file.name
     @fileName.dataset.path = @file.path
 
-    @fileName.classList.add(FileIcons.getService().iconClassForPath(@file.path))
+    iconClass = FileIcons.getService().iconClassForPath(@file.path, this)
```

Comment:
Interesting, I wonder if we should put that setting on the individual packages instead of customizing every package UI via `File-Icons`. Both tree-view and tabs could expose a `disableCustomIcons` setting which, if enabled, wouldn't call the file icons service at all, keeping its default icons.

What do you think? Seems like this would be more customizable and would allow `File-Icons` to not implement a setting for every package that uses it. 💭